### PR TITLE
github/workflow: fix the protoc CI failure

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

https://github.com/confidential-containers/attestation-agent/pull/71#issuecomment-1212880168

### CODE REVIEW

**Comment by vbatts** [comment]
cc @jialez0

File: .github/workflows/basic.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ jobs:
           override: true
           components: rustfmt, clippy
 
+      - name: Install protoc
+        run: |
+          apt-get update && apt-get install -y protobuf-compiler
```
Comment (dubek) [inline .github/workflows/basic.yml:30]: You might need `sudo` in front of these two commands.

Comment (vbatts) [reply .github/workflows/basic.yml:30]: looks like it (with the `permission denied`'s). I had emulated https://github.com/confidential-containers/attestation-agent/blob/main/.github/workflows/eaa_kbc.yml#L36 which does not call out `sudo` :shrug:

**Comment by vbatts** [comment]
upadted, PTAL

**Review by dubek — APPROVED** [review]
LGTM as long as the CI run passes (I'm not authorized to approve it)

**Comment by jialez0** [comment]
I create a new PR #74  to resove the `derive_partial_eq_without_eq` CI clippy check failed.

**Review by jodh-intel — APPROVED** [review]
Thanks @vbatts.

**Review by jialez0 — APPROVED** [review]

**Comment by jialez0** [comment]
@vbatts Now #74 is merged, you can rebase this PR and the CI test will not failed anymore :)

**Comment by vbatts** [comment]
rebased

**Review by sameo — APPROVED** [review]

**Pull request merged**